apps/challenges/urls.py

```diff
@@ -21,8 +21,17 @@
         name='get_all_challenges'),
     url(r'^featured/$', views.get_featured_challenges,
         name='get_featured_challenges'),
+    url(r'^challenge/(?P<challenge_pk>[0-9]+)/queue/(?P<queue_name>[\w-]+)/$',
```

Comment:
Can you please change it to `url(r'^(?P<challenge_pk>[0-9]+)/queues/(?P<queue_name>[\w-]+)/$',`